pkg/sentry/socket/hostinet/socket.go

```diff
@@ -406,10 +406,16 @@ func (s *socketOpsCommon) GetSockOpt(t *kernel.Task, level int, name int, optVal
 		}
 	case linux.SOL_TCP:
 		switch name {
-		case linux.TCP_NODELAY:
+		case linux.TCP_NODELAY, linux.TCP_MAXSEG:
 			optlen = sizeofInt32
 		case linux.TCP_INFO:
 			optlen = linux.SizeOfTCPInfo
+			// tcp_info length should equal to user required due to user process may be compiled with lower kernel version
```

Comment:
In gvisor codebase, we wrap comment to 80 characters and must be a proper english sentence (i.e. end with a period (.) and all that)

// Truncate the output buffer to outLen size.

Reply:
OK, I'll fix this in new PR